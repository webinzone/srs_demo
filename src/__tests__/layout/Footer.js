//As this is a simple non interactive component,
//testing is done to check basic component rendering
import React from "react";
import { shallow } from "enzyme";

import Footer from "../../components/layout/Footer";

describe("<Footer/>", () => {
  it("renders without crashing", () => {
    //Shallow isolates and only renders single component, not parent. So that we get pure unit testing
    const component = shallow(<Footer />);

    //This function checks if the snapshot matches the rendered component
    expect(component).toMatchSnapshot();
  });
});
