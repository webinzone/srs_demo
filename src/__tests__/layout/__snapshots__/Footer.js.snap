// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> renders without crashing 1`] = `
<footer
  className="bg-primary text-white mt-5 p-4 text-center"
  style={
    Object {
      "bottom": "0",
      "left": "0",
      "position": "absolute",
      "right": "0",
    }
  }
>
  Sarthak Sarangi @ 2019
</footer>
`;
